static/images/doubleleftarrow.svg

```diff
@@ -0,0 +1,43 @@
+<svg xmlns="http://www.w3.org/2000/svg">
+<svg viewBox="0 0 400000 549" preserveAspectRatio="xMinYMin slice">
```

Comment:
Why the extra `<svg>`?  Couldn't `viewBox` and `preserveAspectRatio` be set on the outer `<svg>`?